### PR TITLE
Install pikachu via pypi, setup and tox updated for python 3.10

### PR DESCRIPTION
**Pull request by Kohulan**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -45,5 +45,8 @@ exclude =
     RanDepict/__init__.py,
     Tests/test_functions.py,
 
-
+[pytest]
```
Comment (OBrink) [inline tox.ini:48]: I would rather not filter the warnings here, especially when they don't make the linter fail. 
Instead, I would prefer fixing the underlying problems or catch the warnings specifically with `warnings.catch_warnings` if we can't fix them for some reason. But this does not need to be done now

Comment (Kohulan) [reply tox.ini:48]: Ok then lets remove that from tox.

**Review by OBrink — APPROVED** [review]
Looks good, thank you!

**Comment by Kohulan** [comment]
After merging maybe we can create a new release? and resolve #16 ?

**Pull request merged**